packages/snaps-execution-environments/src/common/test-utils/testUtils.ts

```diff
@@ -0,0 +1,350 @@
+// @ts-expect-error Walker has no types yet.
```

Comment:
1. If it's not too difficult, I would prefer adding a small type declaration for this.
2. Since the folder is already named `test-utils`, maybe split this file into a couple separate files, e.g.:
    - `TestSnapExecutor` and its dependencies to `executor.ts`.
    - `endowmentHardeningTestFunction` and its dependencies to `endowments.ts`.
    - And so on.

Reply:
Refactored to be split into two suggested files.

Reply:
Related to the request `1.`, I've tried to add types file and declare it, but it doesn't work with AVA and I cannot find some kind of configuration which would help make it work.

I've added `lavatube.d.ts` file with:
```typescript
// eslint-disable-next-line import/unambiguous
declare class LavaTube {
  constructor(callback: unknown, configuration: object);

  walk(subject: unknown): void;
}

declare module '@lavamoat/lavatube' {
  export = LavaTube;
}
```
I tried moving that file around, but it doesn't help as well. IDE recognizes it, build is passing, but AVA test runner do not, it throws an error:
```
error TS7016: Could not find a declaration file for module '@lavamoat/lavatube'. '/snaps-monorepo/node_modules/@lavamoat/lavatube/src/index.js' implicitly has an 'any' type.
```
If you have some ideas I can try.